src/mlpack/methods/softmax_regression/softmax_regression_impl.hpp

```diff
@@ -41,20 +42,20 @@ size_t SoftmaxRegression::Classify(const VecType& point) const
   return size_t(label(0));
 }
 
-template<typename OptimizerType>
+template<typename OptimizerType, typename... CallbackTypes>
 double SoftmaxRegression::Train(const arma::mat& data,
                                 const arma::Row<size_t>& labels,
                                 const size_t numClasses,
-                                OptimizerType optimizer)
+                                OptimizerType optimizer,
+                                CallbackTypes&&... callbacks)
 {
-  SoftmaxRegressionFunction regressor(data, labels, numClasses,
-                                      lambda, fitIntercept);
+  SoftmaxRegressionFunction regressor(data, labels, numClasses,0);
```

Comment:
Not sure why, you don't use `fitIntercept` here?

Reply:
@zoq  ohhhhh  how i miss, this is the reason for the tested that were failing. thanks a lot